### ISSUE TITLE
reordering components fails

### ISSUE THREAD

**Issue by anarcat** [cachethq/Cachet#524]
i can reorder components in the list in `/dashboard/components` but the change never gets saved.

**Comment by jbrooksuk** [comment]
Interesting, what browser, OS and host are you using? Re-ordering of components works on the demo.

**Comment by anarcat** [comment]
chromium on debian jessie, self-hosted.

**Comment by anarcat** [comment]
hmmm... and now it works.

odd.

**Comment by anarcat** [comment]
actually, no, it doesn't work. i can reorder them and it says "components updated" but the changes do not reflect in the user-facing site. when i come back to the dashboard interface, the changes are gone.

**Comment by jbrooksuk** [comment]
Does it work for you in the demo app?

**Comment by anarcat** [comment]
yes, it does (except for #525 of course ;).

**Comment by jbrooksuk** [comment]
Is this still a problem for you? Do you get any errors in the console when you've re-ordered your components? Are you using MySQL?

**Comment by anarcat** [comment]
yes. the console shows a 500 error, and the network inspector in chrome tells me that the resulting page is `Whoops, looks like something went wrong.`

```
POST http://example.net/dashboard/api/components/order 500 (Internal Server Error)
```

i am using mysql.

**Comment by jbrooksuk** [comment]
It's yourself whom has problems with the CSRF tokens, right?

**Comment by anarcat** [comment]
you mean #525? that was me yes. but i didn't have cookies enabled then, now cookies *are* enabled here.

**Comment by jbrooksuk** [comment]
This should've been closed when we merged #599. @anarcat if you're still experiencing this issue, let us know.

**Comment by anarcat** [comment]
i can't confirm this, git pull blew up in my face (see #633).
